src/utils/hooks/api/use-get-prices.tsx

```diff
@@ -42,19 +43,56 @@ const getPricesByTicker = (currencies: CurrencyExt[], prices: Prices) =>
     return { ...acc, [currency.ticker]: prices[coingeckoId] };
   }, {});
 
-const getPrices = async (currencies?: CurrencyExt[]): Promise<Prices | undefined> => {
-  if (!currencies) {
-    return;
-  }
+const getAPIPrices = async (currencies: CurrencyExt[]): Promise<Prices> => {
```

Comment:
```suggestion
const getCoingeckoPrices = async (currencies: CurrencyExt[]): Promise<Prices> => {
```

Reply:
I went for API because we might not always use coingecko as our source for prices.